pep-B.rst

```diff
@@ -156,6 +156,23 @@ Specification
     `Visualization <../install_locations.html>`_
 
 
+    **Meeting conclusion:**
+        Patches of Python, distutils or pip not needed.
+         - split binaries of system-python and normal python
+         - set sys.prefix of python to /usr/local
+         - set sys.prefix of system_python to /usr
+         - require python3 in system_python
+         - include system-python libs in sys.path of python
```

Comment:
Because `python3`'s `sys.prefix` will be changed to `/usr/local`, it won't see the libs in `/usr/lib/...` (which is where `system-python` will store it's libs). Therefore we need to add `/usr/lib/.../python...` to the `python3`'s sys.path.